docs/tempo/website/configuration/_index.md

```diff
@@ -124,13 +124,16 @@ storage:
 
         blocklist_poll: 5m                       # how often to repoll the backend for new blocks
         blocklist_poll_concurrency: 50           # optional. Number of blocks to process in parallel during polling. Default is 50.
-        cache: memcached                         # optional cache configuration
-        memcached:                               # optional memcached configuration
+        cache: memcached                         # optional. Cache configuration
+        background:                              # optional. Background cache configuration
```

Comment:
let's improve this name to include the word "cache"

`background_cache` ?
`cache_buffer` ?

Reply:
Done! I used `background_cache` :)